assets/sass/_pulumi.scss

```diff
@@ -1,22 +1,14 @@
 @import "pulumi-constants";
 @import "pulumi-toc";
 
-// Drop the rounded corners for code snippets. Ideally we would just do this
-// for the code associated with our language chooser. However, there isn't a
-// way to associate the two. (Since the N language code blocks are siblings
-// to the lang chooser tabs.)
-pre.highlight {
-    border-radius: 0 !important;
```

Comment:
Later in the file we set the border-radius to 2px, so this has no effect. The 2px radius looks fine to me with the lang chooser tabs (if the radius was 4px, it wouldn't look good).